packages/mobile/src/paymentRequest/types.ts

```diff
@@ -12,11 +11,10 @@ export interface PaymentRequest {
   uid?: string
   amount: string
   comment?: string
-  timestamp: Date
+  timestamp: number
   requesterAddress: string
   requesteeAddress: string
   requesterE164Number?: string
-  currency: Currency
```

Comment:
I think we should keep this one to avoid breaking backward compat. Unless the current client doesn't read it at all.
But I think it's still important to know the currency of the amount.
In that case I propose we add a `currencyCode` field.

Reply:
The client doesn't read this at all.

So, for the user the payment requests are made using their local currency. We don't store the value in the local currency directly because that would mean that every client would need to know about all exchange types, which seems unnecessary. So, we store the requested value in cUSD instead and then show it to the requestee in their own local currency. If the requestee wants to pay in cEUR we calculate the exchange rate by doing cUSD -> localCurrency -> cEUR, since all clients fetch their exchange rate with both cUSD and cEUR.

This brings some issues where the requested amount might not match the sent amount, especially if the requester uses a volatile currency and/or a lot of time passes between the request and the send. These issues exist from before this work.

If we add a `currencyCode` field it will not be used at all and its value will be always cUSD. I don't see much value in that to be honest, so I'd rather not add it unless you think I'm missing something.

Reply:
Yes, I just think it's good practice to always have an amount with it's `currencyCode` so it's clear what it is.
Even if in this case we're only gonna use cUSD.